src/screens/DiscoveryScreenV2/elements/RecentSearchItem.js

```diff
@@ -48,7 +48,8 @@ const styles = StyleSheet.create({
     flex: 1,
     alignSelf: 'center',
     paddingTop: 6,
-    paddingBottom: 6
+    paddingBottom: 6,
+    color: COLORS.white2
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change here is about adding a `color` property to the style object. This change seems fine as long as `COLORS.white2` is a valid color.
```diff
- paddingBottom: 6
+ paddingBottom: 6,
+ color: COLORS.white2
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->